File: Olebo/View/src/main/java/jdr/exia/MODELPLACEHOLDER.java
package jdr.exia;

import java.awt.Image;

public class MODELPLACEHOLDER {
	
	private static MODELPLACEHOLDER instance;
	
	private ElementPlaceHolder[] elements; // the array containing the different sprites to display 
	
	public MODELPLACEHOLDER() {
		
		elements = new ElementPlaceHolder[1];
		
		elements[0] = new ElementPlaceHolder();
	}
	
	public static MODELPLACEHOLDER getInstance() {
		if(instance == null)
		{
			instance = new MODELPLACEHOLDER();
		}else{}
		return instance;
		}
	
	
	public Image[] toSprites() { // Turns an array of elements to a corresponding
		
		int length = elements.length;
		
		
		
		
		return null;
	}
	
	
}

File: Olebo/src/main/kotlin/jdr/exia/view/MODELPLACEHOLDER.kt
package jdr.exia.view

import java.awt.Image

object MODELPLACEHOLDER {
    private val elements: Array<ElementPlaceHolder> = arrayOf() // the array containing the different sprites to display

    init {
        elements[0] = ElementPlaceHolder
    }

    fun toSprites(): Array<Image>? { // Turns an array of elements to a corresponding
        val length = elements.size

        return null
    }
}
